### PR TITLE
{bio} [foss/2018b] FastQTL 2.184 and libRmath 3.6.0

### PR DESCRIPTION
**Pull request by pescobar**
libRmath is a library provided with the R tarball that is only used as builddep

maybe we should extend the R easyblock so libRmath is built by default?

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
nic425 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/95062d3fd271fa946b1efa61daf86ed4 for a full test report.

File: easybuild/easyconfigs/f/FastQTL/FastQTL-2.184-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    ('zlib', '1.2.11'),
+    ('bzip2', '1.0.6'),
+    ('XZ', '5.2.4'),
+    ('GSL', '2.5'),
+    ('Boost', '1.67.0'),
+]
+
+buildopts = 'INCMATH=$EBROOTLIBRMATH/include LIB_MATH=$EBROOTLIBRMATH/lib64/libRmath.a '
+# we override LIB_BASE to change -lblas to $LIBBLAS (-lopenblas for foss toolchain)
+buildopts += 'LIB_BASE="-lm -lz -lboost_iostreams -lboost_program_options -lgsl $LIBBLAS"'
```
Comment (smoors) [inline easybuild/easyconfigs/f/FastQTL/FastQTL-2.184-foss-2018b.eb]: @pescobar explicitly adding the Boost directory fixes the test for me: 
```
buildopts += 'LIB_BASE="-L $EBROOTBOOST/lib -lm -lz -lboost_iostreams -lboost_program_options -lgsl $LIBBLAS"'
```

**Comment by pescobar** [comment]
@smoors can you test again please?

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nic20 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/0c696474ceb0d75dbde30d38d2f85838 for a full test report.

File: easybuild/easyconfigs/l/libRmath/libRmath-3.6.0-foss-2018b.eb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+prebuildopts = 'cd %(builddir)s/R-%(version)s/src/nmath/standalone &&'
+
+preinstallopts = 'cd %(builddir)s/R-%(version)s/src/nmath/standalone &&'
+
+pretestopts = 'cd %(builddir)s/R-%(version)s/src/nmath/standalone &&'
+runtest = 'check'
+
+sanity_check_paths = {
+    'files': ["lib64/libRmath.so", "lib64/libRmath.a",
```
Comment (smoors) [inline easybuild/easyconfigs/l/libRmath/libRmath-3.6.0-foss-2018b.eb]: @pescobar please replace with:
```
'files': ["lib64/libRmath.%s" % SHLIB_EXT, "lib64/libRmath.a",
```

Comment (pescobar) [reply easybuild/easyconfigs/l/libRmath/libRmath-3.6.0-foss-2018b.eb]: done

**Comment by smoors** [comment]
Going in, thanks @pescobar!

**Pull request merged**